### ISSUE TITLE
CSS files failed WCAG2 compliancy when using the commercial software CSE HTML Validator Pro

### ISSUE THREAD

**Issue by wet-boew-testing** [wet-boew/wet-boew#2784]
Since jigsaw does not appear to be reliable enough to report CSS3 issues according to https://github.com/wet-boew/wet-boew/issues/1158#issuecomment-13745731 .   Other tool like CSE HTML Validator Pro can find the CSS3 issues...

Here is summary of the errors found with CSE on the latest CSS files:
\wet-boew-dist-3.1.2-release-dist\dist\theme-wet-boew\css\theme-ie-min.css (37 errors, 25 warnings)
\wet-boew-dist-3.1.2-release-dist\dist\theme-wet-boew\css\theme-min.css (4 errors, 10 warnings)
\wet-boew-dist-3.1.2-release-dist\dist\theme-wet-boew\css\theme-ns-ie-min.css (0 error, 0 warning)
\wet-boew-dist-3.1.2-release-dist\dist\theme-wet-boew\css\theme-ns-min.css (0 error, 0 warning)
\wet-boew-dist-3.1.2-release-dist\dist\theme-wet-boew\css\theme-serv-ie-min.css (25 errors, 19 warnings) 
\wet-boew-dist-3.1.2-release-dist\dist\theme-wet-boew\css\theme-serv-min.css (4 errors, 10 warnings)
\wet-boew-dist-3.1.2-release-dist\dist\theme-wet-boew\css\theme-sp-pe-ie-min.css (25 errors, 19 warnings)
\wet-boew-dist-3.1.2-release-dist\dist\theme-wet-boew\css\theme-sp-pe-min.css (4 errors, 10 warnings)

**Comment by hsrudnicki** [comment]
@LaurentGoderre @wet-boew-testing @cvrebert @pjackson28 : can we close this issue or otherwise ?

**Comment by langloisga** [comment]
Running this once in a while mignt not be a bad thing...

I just ran CSE HTML Validator Professional v15.03 for wet-boew.min.css
Found 10 Accessibility Error (x10) (WCAG 2.0 (Level A))	A CSS error was found that may cause ambiguities and inconsistent presentations of the page [A, 4.1.1; G134]. Visit http://www.w3.org/TR/WCAG20-TECHS/G134 for more information. Details: Bad or unknown media feature "transform-3d". Browsers should ignore media queries that contain one or more unknown media features. Message repeated 9 times for (line:char): 6:42010, 6:52891, 6:80718, 6:81383, 6:81822, 6:82267, 6:82710, 6:99840, 6:102742.

Found also the following General CSS errors for wet-boew.min.css 
CSS Error (General)	Bad or unknown media feature "transform-3d". Browsers should ignore media queries that contain one or more unknown media features. (line:char) 6:104793	

CSS Error (x5) (General)	The -webkit-linear-gradient() function is invalid. The expected syntax for -webkit-linear-gradient() is -webkit-linear-gradient( [ <angle> | <side-or-corner> ,]? <color-stop-list> ), where <color-stop-list> is <color-stop>{2,} and <color-stop> is "<color> [ <percentage> | <leng

CSS Error (x3) (General)	The value for the "-webkit-background-clip" property is invalid. Perhaps this value should be "padding"? This proprietary property can be "border", "content", "padding", or "text". NOTE: This "-webkit-" prefixed version of the "background-clip" property accepts these synonyms for the standard values: "border" for "border-box", "content" for "content-box", and "padding" for "padding-box". Message repeated 2 times for (line:char): 6:99840, 6:102742.

CSS Error (General)	The "label~" element is not a known element in this selector. Is it misspelled? Is it a class selector that should be preceded by a period?  (line:char) 6:42010

**Comment by cvrebert** [comment]
`transform-3d` is just a hypothetical unprefixed version of https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-webkit-transform-3d

**Comment by LaurentGoderre** [comment]
The CSS validator is slow at time to get updated

**Comment by hsrudnicki** [comment]
Please update this issue's status whether this issue, which opened in 2013, can be closed or otherwise?

**Comment by hsrudnicki** [comment]
Made "Going Stale" and ready to be closed or otherwise (to keep opened please respond)?

**Comment by hsrudnicki** [comment]
Due to no further discussion/updates, this issue can be closed
